sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TablesSharedKeyCredential.java

```diff
@@ -0,0 +1,139 @@
+package com.azure.data.tables;
```

Comment:
There will be some checkstyle failures such as missing MIT header. And adding documentation to public methods/classes.